doc/software.rst

```diff
@@ -0,0 +1,75 @@
+Software
+========
+
+Install the latest version of the library directly from
+`PyPI <https://pypi.python.org/pypi?:action=display&name=luma.lcd>`__
+with::
+
+  $ sudo -H pip install --upgrade luma.lcd
+
+This will normally retrieve all of the dependencies ``luma.lcd`` requires and
+install them automatically.
+
+Installing Dependencies
+-----------------------
+If ``pip`` is unable to automatically install its dependencies you will have to
+add them manually.
+
+``luma.lcd`` relies on the following packages.
```

Comment:
I'm not sure if listing all dependencies here is a good idea because changes in those dependencies are usually not updated in the docs/forgotten etc. It also implies luma.lcd needs all these packages to work, which isn't true, many of them are optional.

Reply:
I think all of them are required for a successful installation.  It is true that ``spidev`` and ``RPi.GPIO`` are only required for Linux installations.  The others are either direct dependencies or are dependencies of dependencies.  That said, I can remove the table and re-word if desired.  If changed here, I would also change within ``luma.oled``.

Reply:
i'd prefer to not list the deps here.

Reply:
Ok, will modify accordingly.